tmcprototype/DishLeafNode/DishLeafNode/DishLeafNode.py

```diff
@@ -270,7 +271,8 @@ def convert_radec_to_azel(self, data):
 
             # Compute Target Coordinates
             target_radec = data[0]
-            desired_target = katpoint.Target(target_radec)
+            print ("target_radec:", target_radec)
```

Comment:
Can this print be removed?

Reply:
Print statement removed. Commit: 1cb8e394dcea0d91b6be489435940b602451797d